app/services/claim_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_dependency 'evss/claims_service'
+
+class ClaimService
+  EVSS_CLAIM_KEYS = %w(openClaims historicalClaims).freeze
+
+  def initialize(user)
+    @user = user
+  end
+
+  def all
+    raw_claims = client.all_claims.body
+    EVSS_CLAIM_KEYS.each_with_object([]) do |key, claims|
+      next unless raw_claims[key]
+      claims << raw_claims[key].map do |raw_claim|
+        find_or_initialize_claim(raw_claim)
+      end
+    end.flatten
+  rescue Faraday::Error::TimeoutError, Timeout::Error
```

Comment:
Interesting. So, we try to get fresh data from EVSS, and failing that, we pull put the last known good data? If we're doing it this way, can we set a short timeout on the client when it's fronting for a user but a long time out when we're just doing a periodic background task?